etc/lfrc.example

```diff
@@ -35,9 +35,9 @@ map O $mimeopen --ask $f
 # This command is called when current file is not a directory. You may want to
 # use either file extensions and/or mime types here. Below uses an editor for
 # text files and a file opener for the rest.
-cmd open ${{
+cmd open &{{
     case $(file --mime-type -Lb $f) in
-        text/*) $EDITOR $fx;;
+        text/*) lf -remote "send $id \$$EDITOR \$fx";;
```

Comment:
It might be nice to leave a comment here, saying 

```
# We use `lf -remote` here to work around https://github.com/gokcehan/lf/issues/988
```

so that we remember to revert the workaround after we fix the underlying problem.

Reply:
Perhaps I misunderstood. Is the only point here that `file --mime-type -Lb $f` takes a non-negligible amount of time? In that case, ignore my previous comment, this is fine.